front/components/assistant/conversation/ConversationMessage.tsx

```diff
@@ -29,7 +34,17 @@ export function ConversationMessage({
         </div>
       </div>
       <div className="flex flex-col items-start gap-2 sm:flex-row">
-        {buttons}
+        {buttons &&
+          buttons.map((button) => (
+            <div key={uuid4()}>
+              <IconButton
+                variant="tertiary"
+                size="xs"
```

Comment:
Based on the Figma don't we want sm?

Reply:
TLDR => for now I switch to sm, and add a backlog item regarding responsive design cleanup pass with ed

I don't see component sizes on the figma but based on text sizes you are right. However I beleive on mobile sm takes up a lot of space. There are also multiple glitches on mobile when the figma is strictly followed on desktop (e.g. horizontal padding cf screenshot) -- AFAICT there is no mobile version of the figma yet. Not necessarily needed yet, but a little cleanup would be good and cheap
![image](https://github.com/dust-tt/dust/assets/5437393/8930ee57-02a7-434c-8277-87e2ccc63e6e)

Reply:
We can def use two different for mobile and non mobile indeed.

re mobile responsiveness it seems that as is the buttons do overflow the layout a bit?

Reply:
Yes that was my intention